### ISSUE TITLE
NodeWatcher issue

### ISSUE THREAD

**Issue by NDari** [jasonmit/ember-intl#347]
Hello,

First, a big thank you for the fantastic work with this addon.

I have ran into an issue recently, where I was attempting to put some translations into my addon to be consumed by users. To do this, I upgraded my version of ember-intl to "~2.10", and put the `translations` folder at the root of my addon.

However, doing this would create a huge amount of (seemingly) indecipherable errors. These errors would go away if I deleted the `translations` folder, or downgraded to "~2.4".

Long story short, the reason was that I was using nodewatcher, and not watchman.

I think it would be invaluable to have some sort of a deprecation warning so that nobody would have to go through the hours of debugging I went through. 

Is this something that is of value? I am not sure how much effort this would require.

**Comment by jasonmit** [comment]
ember-intl didn't deprecate nodewatcher over watchman.  I personally haven't run into this and it's the first time this is reported.  I'd be happy to investigate, to see if I can optimize for nodewatcher.

**Comment by NDari** [comment]
I will need to do a few things since I have installed watchman and no longer have the issue. I will try to recreate it tonight (USA time)

**Comment by leighwilliams** [comment]
Hi,

I'm seeing this issue since upgrading from 2.5.0 to 2.11.1.

Ubuntu 15.10.

I've tried ember-cli 2.4.3 with node **0.12.7** and node **5.10.1** with the same result.
```
❯ ember s
Could not start watchman; falling back to NodeWatcher for file system events.
Visit http://ember-cli.com/user-guide/#watchman for more info.
Livereload server on http://localhost:49152
Serving on http://localhost:4200/
file added tmp/broccoli_merge_trees-input_base_path-rgzKJZD0.tmp/0
...
file added tmp/sass_compiler-output_path-u41TGrNC.tmp
fs.js:1172
    throw errnoException(err, 'watch');
          ^
Error: watch ENOENT
    at exports._errnoException (util.js:746:11)
    at FSWatcher.start (fs.js:1172:11)
    at Object.fs.watch (fs.js:1198:11)
    at NodeWatcher.watchdir (<ADDON PATH>/node_modules/sane/src/node_watcher.js:144:20)
    at NodeWatcher.<anonymous> (<ADDON PATH>/node_modules/sane/src/node_watcher.js:278:14)
    at FSReqWrap.oncomplete (fs.js:95:15)
```

Removing the translations folder at addon root also resolves the issue for me (it only contains 1 file), as does using Watchman.

I can reproduce this on a fresh _addon_ (ember-cli 2.4.3) and `ember install ember-intl`.

**Comment by jasonmit** [comment]
> Removing the translations folder at addon root

This is happening while developing your addon?

**Comment by leighwilliams** [comment]
That is correct.

**Comment by NDari** [comment]
Sorry for the HUGE delay... 

I just wanted to chime in that I got the same error when I reported this issue, and the same remedies (removing the translation folder at the root of the addon or using watchman) fixed my issues.

**Comment by jasonmit** [comment]
@leighwilliams @NDari could either of you reproduce this with an example addon (or working addon) and push it to github somewhere for me to iterate on.  I haven't yet been able to find the sweet spot to reproduce it so I can begin to triage it.

Thanks 👍

**Comment by leighwilliams** [comment]
Apologies for the delay, I'm seeing a some strange behaviour and I'm trying not to give false or misleading info.

I can reproduce it on a completely new addon with ember-intl installed. See [https://github.com/leighwilliams/ember-intl-testing](https://github.com/leighwilliams/ember-intl-testing). 

It seems to surface on Linux before is does on Mac. This particular repo runs fine on a Mac, but I _have_ seen this issue on a Mac with a real project.

Additionally, this cropped up today for me on our dev server in an ember app.

I'm trying to reproduce it on a Mac with something shareable, I'll provide that repo if so.

Thanks.

**Comment by jasonmit** [comment]
I wasn't able to produce on Ubuntu, but will keep trying.  Once I am able to reproduce I'll be able to fix it, until then I may need to keep pinging you.  I know it's isolated around the addon support that went in.

Can you try reverting this commit to see if the issue goes away:
https://github.com/jasonmit/ember-intl/commit/3066a497e813d4bd28e07318d0d7f7d18e24fd4d

**Comment by leighwilliams** [comment]
> until then I may need to keep pinging you.

No worries. :+1:

> Can you try reverting this commit to see if the issue goes away:
3066a49

No luck there.

> I know it's isolated around the addon support that went in.

Indeed. Going from 2.7 to 2.8 is where it's happening.

**Comment by gcusnieux** [comment]
Hello, 
I have same problem on win7, it's okay on unix machine.
Do you have an idea, for win7 users ?

**Comment by gcusnieux** [comment]
@leighwilliams  Try https://github.com/jasonmit/ember-intl/pull/355.

**Comment by leighwilliams** [comment]
@gcusnieux Yes, that has resolved the issue I was having. Thanks!
